t04_refactored.py

```diff
@@ -225,7 +225,44 @@ def team_18_adv():
 ###################################################################################
 
 def team_19_adv():
+    from time import sleep
```

Comment:
Uncommon to have imports inside of functions, but this is fine for this assignment (they should usually be the first lines after the header block)